FCC Survey Page/css/sp-css.css

```diff
@@ -0,0 +1,113 @@
+@import url('https://fonts.googleapis.com/css?family=Noto+Sans+JP|Roboto+Slab&display=swap');
+
+html,
+body {
+    background-color: #A9D7D1;
+}
+
+#form-outer {
+    border: 5px;
+    border-radius: 10px;
+    max-width: 65%;
+    margin: 0 auto;
+    background-color: #FAFAFA;
+    padding: 0.5em;
+}
+
+#title {
+    font-family: 'Noto Sans JP', sans-serif;
+    font-size: 2em;
+    text-align: center;
+ }
+
+#description {
+    margin: 1.2em 0 1.2em 0;
+    padding: 0.5em;
+    border: 4px;
+    border-radius: 17px;
+    background-color: #fcca5d;
+    font-size: 1.2em;
+    font-family: 'Roboto Slab', serif;
+    text-align: center;
+}
+
+input#name, input#email, input#age {
+    height: 20px;
+    width: 50%;
+    border: 2px solid rgb(161, 135, 79);
+    border-radius: 2px;
+}
+
+select#exp {
+    height: 2em;
+    width: 8em;
+    border: 2px solid rgb(161, 135, 79);
+    border-radius: 20px;
+}
+
+.row {
+    margin: 0.7em;
+    font-family: 'Noto Sans JP', sans-serif;
+}
+
+ul {
+    margin-block-start: auto;
+    padding-left: unset;
+    list-style: none;
+}
+
+input#other {
+    height: 20px;
+    border: 2px solid rgb(161, 135, 79);
+    border-radius: 2px;
+}
+
+textarea#comment {
```

Comment:
**Important:** Please put `max-width: 100%;`. It is recommended for textareas as you can resize them by default. If the user decide to resize the textarea out of the layout, it would break the initial design. You can also put `max-height: 500px;` (for example). To restrict the resizing from height also.